ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/MLAG_OSPF_TESTS.yml

```diff
@@ -0,0 +1,34 @@
+---
+underlay_routing_protocol: ospf
+mlag_support: true
+
+type: l3leaf
+
+l3leaf:
+  defaults:
+    mlag_peer_vlan: 4094
+    mlag_peer_l3_vlan: 4094
+    mlag_peer_ipv4_pool: 10.10.255.0/24
+    uplink_ipv4_pool: 10.10.101.0/24
+    loopback_ipv4_pool: 192.168.255.0/24
+    loopback_ipv4_offset: 32
+    vtep_loopback_ipv4_pool: 192.168.254.0/24
+    virtual_router_mac_address: 00:1c:73:00:DC:01
+    platform: vEOS-LAB
+    uplink_switches: ["DC1-SPINE1"]
+    uplink_interfaces: ["Ethernet1"]
+  node_groups:
+    MLAG_OSPF_L3LEAF1:
+      nodes:
+        MLAG-OSPF-L3LEAF1A:
+          id: 4
+          bgp_as: 65161
+          mgmt_ip: 192.168.201.114/24
+          uplink_switch_interfaces: [Ethernet16]
```

Comment:
`Ethernet16` has been taken on Spine1 for some recently merged PR:
https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml#L228

(and so has `Ethernet17`) - this causes one of the molecule tests to fail in CI

Could you use `Ethernet18`and push back the PR?

Reply:
thanks @gmuloc - resolved in [9be9059](https://github.com/aristanetworks/ansible-avd/pull/1842/commits/9be90596c9ed5d89aff545bd28d461c0e01c1c31)